CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ Read more about the session argument for computed fields in the [docs](https://h
 
 - console: add read replica support section to pro popup (#4118)
 - cli: list all avialable commands in root command help (fix #4623)
+- console: modify default value for PK (fix #4075) (#4679)
```

Comment:
Could you also move it one line up? So that all `console:` entries are grouped together.
```suggestion
- console: allow modifying default value for PK (fix #4075) (#4679)
```